addons/skin.estuary/xml/Variables.xml

```diff
@@ -372,6 +375,12 @@
 		<value condition="!String.IsEmpty(Window.Property(IsRadio))">$LOCALIZE[19199] - $LOCALIZE[19024]</value>
 		<value>$LOCALIZE[19199] - $LOCALIZE[19023]</value>
 	</variable>
+	<variable name="RDSInfoSubTitle">
+		<value condition="String.IsEmpty(MusicPlayer.ChannelName)">$INFO[Player.Title]</value>
+		<value condition="String.IsEmpty(Player.Title)">$INFO[MusicPlayer.ChannelName]</value>
+		<value condition="String.IsEmpty(MusicPlayer.ChannelName) + String.IsEmpty(Player.Title)"></value>
+		<value>$INFO[MusicPlayer.ChannelName] - $INFO[Player.Title]</value>
```

Comment:
the three lines above can be merged into one:
`<value>$INFO[MusicPlayer.ChannelName]$INFO[Player.Title, - ]</value>`

Reply:
Hehe, ofc. Thanks.